src/amo/css/App.scss

```diff
@@ -27,7 +27,24 @@ body {
 }
 
 .App-content {
-  padding: 0 0 10px;
+  background: $accent-color;
+
+  > div {
+    padding: 10px;
+  }
+
+  > .full-width {
```

Comment:
These override classes make me think the padding shouldn't be on `.App-content`.